riscv/src/large_field/code_gen.rs

```diff
@@ -197,7 +197,7 @@ machine Main with min_degree: {}, max_degree: {} {{
 
 {}
 
-let initial_memory: (fe, fe)[] = [
```

Comment:
this needs to be `(int, int)` because we cant represent `u32` values using `fe` in small field